### PR TITLE
Inputs Completed

### PR DESCRIPTION
**Pull request by MayurChawla**
We have multiple input type : 
- textbox
- password
- with validation message
- TextArea

### CODE REVIEW

File: Components/Inputs/Inputs.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    --secondary-color : #eb679a;
+    --light-color : #ffeff5;
+    --dark-color : #72022d;
+
+    --badge-primary : #0d6efd;
+    --badge-success : #198754;
+    --badge-warning : #ffc107;
+    --badge-link : #fd7e14;
+    --badge-info : #0dcaf0;
+    --badge-inside-button : goldenrod;
```
Comment (aurobindoGupta) [inline Components/Inputs/Inputs.css:14]: put these in a root file that is used every where

**Review by GuptaEesh — COMMENTED** [review]
Code looks clean, good CSS variables' naming. Good to merge

File: Components/Inputs/Inputs.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+* {
+    margin: 0;
+    padding: 0;
```
Comment (aurobindoGupta) [inline Components/Inputs/Inputs.css:18]: use root file for reset files

**Review by aurobindoGupta — APPROVED** [review]
nice

File: Components/Inputs/Inputs.css (hunk 2 above)
Comment (MayurChawla) [reply Components/Inputs/Inputs.css:18]: yes will do that post achieving current functionality

**Pull request merged**